### PR TITLE
Update _index.md

### PR DESCRIPTION
**Pull request by glarocca**
Revised text and added missing href links

# Summary

---

**Related issue :**

### CODE REVIEW

File: content/en/users/cloud-compute/ec3/_index.md [hunk 1]
```diff
@@ -8,9 +8,20 @@ description: >
...
 automatically resized to fit the load (e.g. number of jobs at the batch system).
 
-This section of the documentation will guide you on how to use EC3 with EGI Cloud
-providers using the OpenStack API and OpenID Connect for deploying your clusters.
+The following section of the documentation will guide you on how to:
+
+-   Request access to EC3,
+-   Deploy simple EC3 elastic cluster on top of the IaaS providers of the EGI Cloud,
+    either using the web interface or the command-line interface,
+-   Run pre-configured scientific applications in the EC3 elastic cluster.
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/_index.md:26]:
```suggestion
- Request access to EC3,
- Deploy simple EC3 elastic cluster on top of the IaaS providers of the EGI Cloud,
   either using the web interface or the command-line interface,
- Run pre-configured scientific applications in the EC3 elastic cluster.
```

Comment (gwarf) [reply content/en/users/cloud-compute/ec3/_index.md:26]: Fixing the spacing of the list item

File: content/en/users/cloud-compute/ec3/_index.md [hunk 2]
```diff
@@ -8,9 +8,20 @@ description: >
 
 [EC3](http://servproject.i3m.upv.es/ec3/) (Elastic Cloud Compute Cluster) is a
 tool to create elastic virtual clusters on Infrastructure as a Service
-(IaaS) providers. It supports the provisioning of clusters running TORQUE,
-SLURM, SGE, HTCondor, Mesos, Nomad, Kubernetes and others that will be
+(IaaS) providers, either public (such as [Amazon Web Services](https://aws.amazon.com/), 
+[Google Cloud](http://cloud.google.com/) or [Microsoft Azure](http://azure.microsoft.com/)) 
+or on-premises (such as [OpenNebula](http://www.opennebula.org/) and 
+[OpenStack](http://www.openstack.org/)).
+It supports the provisioning of clusters running [TORQUE](http://www.adaptivecomputing.com/products/open-source/torque),
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/_index.md:15]:
```suggestion
It supports the provisioning of clusters running
[TORQUE](http://www.adaptivecomputing.com/products/open-source/torque),
```

File: content/en/users/cloud-compute/ec3/_index.md [hunk 3]
```diff
@@ -8,9 +8,21 @@ description: >
...
 
-This section of the documentation will guide you on how to use EC3 with EGI Cloud
-providers using the OpenStack API and OpenID Connect for deploying your clusters.
+The following section of the documentation will guide you on how to:
+
+- Request access to EC3,
+- Deploy simple EC3 elastic cluster on top of the IaaS providers of the EGI Cloud,
+  either using the web interface or the command-line interface,
+- Run pre-configured scientific applications in the EC3 elastic cluster.
+
```
Comment (gwarf) [inline content/en/users/cloud-compute/ec3/_index.md:28]:
```suggestion
```

File: content/en/users/cloud-compute/ec3/_index.md [hunk 4]
```diff
@@ -8,9 +8,20 @@ description: >
...
+[Kubernetes](https://kubernetes.io/) and others that will be
 automatically resized to fit the load (e.g. number of jobs at the batch system).
 
-This section of the documentation will guide you on how to use EC3 with EGI Cloud
-providers using the OpenStack API and OpenID Connect for deploying your clusters.
+The following section of the documentation will guide you on how to:
+
+- Request access to EC3,
+- Deploy simple EC3 elastic cluster on top of the IaaS providers of the EGI Cloud,
+  either using the web interface or the command-line interface,
```
Comment (enolfc) [inline content/en/users/cloud-compute/ec3/_index.md:26]: do we have the web interface part? Maybe we should add that before

Comment (glarocca) [reply content/en/users/cloud-compute/ec3/_index.md:26]: That's the plan. Add a new section as reported in this doc: https://docs.google.com/document/d/14YZ2IQDU1p2gE0ofEfk_mJKUHFqnbJG7qv17mT-ki_c/edit#

Comment (enolfc) [reply content/en/users/cloud-compute/ec3/_index.md:26]: Then let's go in the right order, I would add those docs as part of this PR and we merge everything together

Comment (glarocca) [reply content/en/users/cloud-compute/ec3/_index.md:26]: Ok, fine with me.

**Review by gwarf — CHANGES_REQUESTED** [review]
Thanks!
Please fix the TORQUE link that is broken: https://github.com/EGI-Foundation/documentation/pull/126/checks?check_run_id=1357513000#step:4:37

File: content/en/users/cloud-compute/ec3/_index.md (hunk 4 above)
Comment (gwarf) [reply content/en/users/cloud-compute/ec3/_index.md:26]: So if this PR was made with the online editor, I'm not sure how possible/easy it's to add other files to this PR.
May have to do checkout of the PR locally to a branch and push it, but I didn't test and document this...

Comment (enolfc) [reply content/en/users/cloud-compute/ec3/_index.md:26]: I did this once for another repo, as long as you keep committing changes to the same branch, things will be added here.

Comment (gwarf) [reply content/en/users/cloud-compute/ec3/_index.md:26]: So in fact it's easy with [GitHub CLI](https://docs.egi.eu/about/git/#starting-with-the-github-cli):
```sh
gh pr checkout 126
vim ...
git add ...
git commit ... -m xxx
git push
```

And it should just work!

**Comment by gwarf** [comment]
> Thanks!
> Please fix the TORQUE link that is broken: https://github.com/EGI-Foundation/documentation/pull/126/checks?check_run_id=1357513000#step:4:37

@glarocca should we replace it by https://github.com/adaptivecomputing/torque ?
Even  their GitHub repo is pointing to that page that is now broken :/

**Review by gwarf — APPROVED** [review]

**Comment by gwarf** [comment]
@glarocca I directly updated the file to fix the issues, should we wait for some more files or can we commit this already?

**Pull request merged**